server/api/cart.js

```diff
@@ -27,21 +27,22 @@ router.get('/basket/:id', async (req, res, next) => {
 
 router.post('/add', async (req, res, next) => {
   try {
-    //receives price, quantity, cartId & albumId and adds item from cart
+    //receives price, quantity, userId & albumId and adds item from cart
     const price = req.body.price;
     const qty = req.body.quantity;
     const albumId = req.body.albumId;
-    const cartId = req.body.cartId;
-    const cart = await Cart.findByPk(cartId);
-    const item = await cart.addAlbum(albumId);
-    await item.update({ price: price, quantity: qty });
+    const userId = req.body.userId;
+    const cart = await Cart.findAll({ where: { userId: userId } });
```

Comment:
cart is returning as an array because we used .findAll instead of findByPk
findAll returns an array of all data that matches the where option, and findByPk returns an object of the only row that matches the ID you input